.markdownlint.yaml

```diff
@@ -0,0 +1,24 @@
+default: true
+
+MD010:
```

Comment:
Nit: can we use comment to inform what each rule is, so that I don't have to google the library for its meaning?

Reply:
good idea, just added the comment with the rule url as well